apps/test-suite/components/Suites.js

```diff
@@ -0,0 +1,38 @@
+import Constants from 'expo-constants';
+import React from 'react';
+import { FlatList, StyleSheet } from 'react-native';
+
+import DoneText from './DoneText';
+import SuiteResult from './SuiteResult';
+
+export default function Suites({ suites, ...props }) {
+  const ref = React.useRef(null);
+
+  function scrollToEnd() {
+    if (ref.current) ref.current.scrollToEnd({ animated: false });
+  }
+
+  return (
+    <FlatList
+      ref={ref}
+      style={styles.list}
+      contentContainerStyle={styles.contentContainerStyle}
+      data={[...suites]}
+      keyExtractor={item => item.get('result').get('id')}
+      renderItem={({ item }) => <SuiteResult r={item} depth={0} />}
+      ListFooterComponent={() => <DoneText {...props} />}
```

Comment:
It is not intuitive that the props on the Suites component will be applied to the footer -- typically they'd be applied to the root component (the FlatList). That said, the props that are passed into the Suites component all do make sense, and it's just an implementation detail that they are passed through to the footer. To communicate this better in the code, write out the props explicitly:

```
// export default function Suites({ suites, ...props }) {
export default function Suites({ suites, done, results, numFailed }) {

// ListFooterComponent={() => <DoneText {...props} />}
ListFooterComponent={() => <DoneText done={done} results={results} numFailed={numFailed} />}
```

This communicates that the intent is not to proxy through all the props of the Suites component to the footer, but to pass through the done+results+numFailed props specifically. The fact that "...props" happens to just be done+results+numFailed is something that could change over time and break this code.